recipes/ninja/all/test_package/conanfile.py

```diff
@@ -3,11 +3,10 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
-    generators = "VirtualBuildEnv"
     test_type = "explicit"
 
     def build_requirements(self):
-        self.tool_requires(self.tested_reference_str)
+        self.requires(self.tested_reference_str)
```

Comment:
@jcar87 what's the meaning of calling  `self.requires` in `build_requirements(self)` ?